modules/hello-world/examples/sample-app.py

```diff
@@ -0,0 +1,478 @@
+# REFERENCE FILE ONLY.
+# This file is a copy of 'travel.py' in the 'try-cb-python' repo. It's not
```

Comment:
This file will need to become a 7.2 version of https://github.com/couchbaselabs/try-cb-python/blob/7.1/travel.py - we'll need a mechanism to keep them in sync - but I'm not going to block on that, so we'll call it a future todo.